documentation/recommendations.adoc

```diff
@@ -799,30 +797,95 @@ RETURN m.title, SUM( pearson * r.rating) AS score
 ORDER BY score DESC LIMIT 25
 ----
 
-
-.kNN movie recommendation using Pearson similarity function
-[source, cypher]
-----
-MATCH (u1:User {name: 'Cynthia Freeman'})-[x:RATED]->(movie:Movie)
-WITH u1, gds.alpha.similarity.asVector(movie, x.rating) AS u1Vector
-MATCH (u2:User)-[x2:RATED]->(movie:Movie) WHERE u2 <> u1
-
-WITH u1, u2, u1Vector, gds.alpha.similarity.asVector(movie, x2.rating) AS u2Vector
-WHERE size(apoc.coll.intersection([v in u1Vector | v.category], [v in u2Vector | v.category])) > 10
-
-WITH u1, u2,  gds.alpha.similarity.pearson(u1Vector, u2Vector, {vectorType: "maps"}) AS similarity
-ORDER BY similarity DESC
-LIMIT 10
-
-MATCH (u2)-[r:RATED]->(m:Movie) WHERE NOT EXISTS( (u1)-[:RATED]->(m) )
-RETURN m.title, SUM( similarity * r.rating) AS score
-ORDER BY score DESC LIMIT 25
-----
-
 ++++
 </div>
 ++++
 
```

Comment:
you could have had it easier with 4 slashes before and after the block :) `////`